typing/mtype.ml

```diff
@@ -30,34 +30,42 @@ let rec scrape env mty =
       end
   | _ -> mty
 
+let scrape_lazy env mty =
+  let open Subst.Lazy in
+  match mty with
+    MtyL_ident p -> of_modtype (scrape env (Mty_ident p))
```

Comment:
This seems to find the full signature of `p`, could we not use `find_modtype_expansion_lazy` instead?

Reply:
Good idea.